schema.sql

```diff
@@ -8,4 +8,18 @@ CREATE TABLE animals (
 );
 
 -- Vet clinic database: query and update animals table
-ALTER TABLE animals ADD COLUMN species VARCHAR;
\ No newline at end of file
+ALTER TABLE animals ADD COLUMN species VARCHAR;
+
+-- Vet clinic database: query multiple tables
+
+ CREATE TABLE owners(id INT GENERATED ALWAYS AS IDENTITY, full_name VARCHAR, age INT, PRIMARY KEY(id));

```

Comment:
- [ ] In PostgreSQL, if you declare a VARCHAR column without specifying a maximum length, it will have a default length of 1. 
- This behavior will not be very useful in the case of your project, as it limits the column to only store single-character strings by default. It's generally recommended to explicitly specify the maximum length for VARCHAR columns to match the expected data size for your application.

### Sample:
`full_name varchar(255),`

Reply:
Hi @Shedrack-Sunday, 
Thank you for your review. I have made the necessary correction and updated my files. Below are screenshots.
![image](https://github.com/MargaretTechworld/Vet-Clinic-Database/assets/60655304/2dc1298e-9a23-42e1-85ba-c28a948f5390)
![image](https://github.com/MargaretTechworld/Vet-Clinic-Database/assets/60655304/2366394e-094a-4080-a417-a0b9bdcd458b)
![image](https://github.com/MargaretTechworld/Vet-Clinic-Database/assets/60655304/436155b7-2191-45c7-9071-b3fc325636e1)